powerauth-nextstep/pom.xml

```diff
@@ -188,10 +167,6 @@
                             <groupId>org.projectlombok</groupId>
                             <artifactId>lombok</artifactId>
                         </exclude>
-                        <exclude>
```

Comment:
Why are you removing this? It excludes `servlet-api` from `lib-provided` and `java -jar` should still run, shouldn't it?